### DISCUSSION TITLE
fatal: [console.sb]: FAILED! => {"changed": false, "msg": "Could not find or access '/root/mosip-infra/deployment/sandbox-v2/../v3/mosip/kernel/masterdata' on the Ansible Controller.\nIf you are using a module and expect the file to exist on the remote, see the remote_src option"}

### DISCUSSION THREAD

**Discussion by itproguide** [mosip/mosip#5] [Q&A] [answered]
TASK [Clone masterdata repo] ***************************************************************************************************************************************************************

TASK [github : git checkout] ***************************************************************************************************************************************************************
fatal: [console.sb]: FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result", "changed": true}
...ignoring

TASK [copy masterdata script directory from v3 to tmp directory] ***************************************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: If you are using a module and expect the file to exist on the remote, see the remote_src option
fatal: [console.sb]: FAILED! => {"changed": false, "msg": "Could not find or access '/root/mosip-infra/deployment/sandbox-v2/../v3/mosip/kernel/masterdata' on the Ansible Controller.\nIf you are using a module and expect the file to exist on the remote, see the remote_src option"}

PLAY RECAP *********************************************************************************************************************************************************************************
console.sb                 : ok=252  changed=75   unreachable=0    failed=1    skipped=57   rescued=0    ignored=4   
dmzmaster.sb               : ok=49   changed=12   unreachable=0    failed=0    skipped=30   rescued=0    ignored=0   
dmzworker0.sb              : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzmaster.sb                : ok=49   changed=12   unreachable=0    failed=0    skipped=30   rescued=0    ignored=0   
mzworker0.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker1.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker2.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker3.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker4.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker5.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker6.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker7.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   
mzworker8.sb               : ok=38   changed=7    unreachable=0    failed=0    skipped=28   rescued=0    ignored=0   

[root@console sandbox-v2]# cd /root/mosip-infra/deployment/sandbox-v2/../v3/mosip/kernel/masterdata
-bash: cd: /root/mosip-infra/deployment/sandbox-v2/../v3/mosip/kernel/masterdata: No such file or directory

It seems the master data file is missing 
Any help appreciated. Thank you.

**Comment by nayakrounak** [comment, marked as answer]
Hi @itproguide, 

Can you reach out to us on our community channel: https://community.mosip.io/ ?

Thanks and Regards,
Rounak
